pkg/apis/servicecatalog/types.go

```diff
@@ -365,10 +372,20 @@ type BindingSpec struct {
 	// Immutable.
 	InstanceRef v1.LocalObjectReference
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is a set of the parameters to be
 	// passed to the underlying broker.
+	// The inline YAML/JSON payload to be translated into equivalent
+	// JSON object.
```

Comment:
@pmorie yep, that's correct, but you can't declare duplicate property names within a single JSON anyway.
We might have some wording around conflicts between `parameters` and `parametersFrom` though.